### PR TITLE
[INSTORE-625] Convert MSTest to NUnit.

### PR DESCRIPTION
**Pull request by marisa-ashour**
**Ticket**:
https://levelup.atlassian.net/browse/INSTORE-625

**Test**:
Verified all of the tests had the same results pre and post-conversion.

**Notes**:
- So, coveralls percentages are slightly different (from 100% to 98%) . This is because we are no longer calling the `ToString()` override for `AdjustedCheckValues`. However - we weren't 'testing' this in the original MSTest version, we were just using them as part of our Assert statements. So effectively we still have not lost testing coverage %, the old number wasn't accurate. 
- I have consolidated the tests that are duplicates / edge cases of one another to use `TestCase`, instead of copying/pasting code. However, I've tried to keep test cases that have the same logic but different purposes separate. (For example, if a test has mostly identical logic, but one is testing an order that is fully exempt, and one is testing an order that is not exempt at all, I left them as two separate tests). 
- I'm more than willing to back the consolidated TestCases back into their own individual tests.

### CODE REVIEW

File: LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs [hunk 1]
```diff
@@ -68,23 +69,25 @@ public void SplitTenderExample_LevelUp_Then_Cash()
             // tax amount unchanged
             // spend amount unchanged
             // exemption amount unchanged
-
+                       
             AdjustedCheckValues expectedCompletedOrderValues =
                 new AdjustedCheckValues(expectedSpendAmount, expectedTaxAmount, expectedExemptionAmount);
 
+            // Arrange
```
Comment (navidmatin) [inline LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs]: I don't think this `//Arrange` is necessary since there is one on line 61.

File: LevelUp.Pos.ProposedOrders.Tests/UpdateExemptionAmountTests.cs [hunk 2]
```diff
@@ -16,340 +16,186 @@
...
 {
-    [TestClass]
+    [TestFixture]
     public class UpdateExemptionAmountTests
     {
-        #region Partial Payments
+        // Too much is exempt
+        [Test]
+        [TestCase(1200, 900)] // Significantly over total
+        [TestCase(1050, 900)] // Slightly over total
```
Comment (navidmatin) [inline LevelUp.Pos.ProposedOrders.Tests/UpdateExemptionAmountTests.cs]: Nitpick: Can we change this test case to be only 1 cent over the total.

File: LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs (hunk 1 above)
Comment (marisa-ashour) [reply LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs]: True... Some of these tests were weird since there are two parts to the test - proposed / completed
